src/components/Nav/_nav-item.scss

```diff
@@ -0,0 +1,181 @@
+// TODO refactor to reduce nesting
+// stylelint-disable max-nesting-depth
+.nav__item {
+  &.nav__item {
+    margin: 0;
+    padding: 0;
+    position: static;
+
+    &:before {
+      content: none;
+    }
+  }
+
+  .first {
+    display: none;
+  }
+
+  // TODO: update breakpoint mixin to allow max-width which also removes a pixel
```

Comment:
I'm working on a local branch to implement https://github.com/wellcometrust/corporate/issues/5829 which I think would be a big help for us

Reply:
👍